pincer/commands/arg_types.py

```diff
@@ -42,6 +51,12 @@ class CommandArg(metaclass=_CommandTypeMeta):
     def __init__(self, command_type, *args):
         self.command_type = command_type
         self.modifiers = args
+        _log.warn(
+            "CommandArg is deprecated and will be removed in future releases."
+            " `typing.Annotated`/`typing_extensions.Annotated.` should be used instead."
+            " See https://docs.pincer.dev/en/latest/interactions.html#arguments for"
```

Comment:
Link to the stable docs. We want it to correspond to the release rather than the master branch.

```suggestion
            " See https://docs.pincer.dev/en/stable/interactions.html#arguments for"
```